Tests/DatadogTests/Datadog/RUM/RUMMonitor/Scopes/RUMSessionScopeTests.swift

```diff
@@ -237,186 +237,13 @@ class RUMSessionScopeTests: XCTestCase {
 
         // When
         let commandTime = sessionStartTime.addingTimeInterval(1)
-        let command = RUMCommandMock(time: commandTime, canStartBackgroundView: .mockRandom(), canStartApplicationLaunchView: false)
+        let command = RUMCommandMock(time: commandTime, canStartBackgroundView: .mockRandom())
         XCTAssertTrue(scope.process(command: command, context: context, writer: writer))
 
         // Then
         XCTAssertTrue(scope.viewScopes.isEmpty, "It should not start any view scope")
     }
 
-    // MARK: - Application Launch Events Tracking
```

Comment:
Most of those tests are still relevant, no?

Reply:
I don't think so? Most of these deal with automatically creating the ApplicationLaunch view on commands that can do so, (the removed `canStartApplicationLaunchView` property) and should be covered by the new tests instead.

Looking through, I think I am missing a test for not creating the ApplicaitonLaunch view if it's not the initial session or booted after the background view.

Reply:
I think the `testGivenAppInBackgroundAndBETDisabled_whenReceivingAnyCommand_itDoesNotCreateAnyScope` can be kept.

Reply:
Probably the BET enabled one as well. I might move them into where the other ApplicaiotLaunch events are tracked as well? Yes?

Reply:
Sounds good, yes 👍

Reply:
Yes, `testWhenConfigurationSourceIsSet_applicationStartUsesTheConfigurationSource` definitely shouldn't have been removed. Good catch!

The other tests I think are no longer correct, but definitely please let me know if you disagree. New logic here is still a bit confusing.

`testWhenNoLoadingTime_itSendsApplicationStartAction_basedOnLoadingDate` Is no longer correct as "LoadingTime" (actually `launchTime`) is the time until `applicationDidBecomeActive`, whereas we now time until first view.

`testWhenActivePrewarm_itSendsApplicationStartAction_withoutLoadingTime` is now covered by `testWhenAppLaunchesWithActivePrewarm_itDoesNotSendApplicaitonStartDuration` (in RUMMonitorTests)